components/organisms/cards/CardComponent.vue

```diff
@@ -6,8 +6,10 @@
             class="border-0 cards-card"
             no-body
         >
-            <b-card-body class="card-body onvirtual-card overflow-hidden shadow-hover-sm">
-                <div class="ratio-card overflow-hidden">
+            <b-card-body
+                class="card-body onvirtual-card overflow-hidden shadow-hover-sm ratio ratio-card"
+            >
+                <div>
```

Comment:
do we need this div now ?

Reply:
Yes, tried removing it and the card breaks